applications/NXmx.nxdl.xml

```diff
@@ -142,6 +142,43 @@
                         <dim index="4" value="k" required="false"/>
                     </dimensions>
                 </field>
+
+                <field name="data_scaling_factor" type="NX_FLOAT" optional="true">
+                    <doc>
+                        An optional scaling factor to apply to the values in data.
+
+                        The elements in data are often stored as integers for efficiency reasons and need
+                        further correction, generating floats.  The two fields data_scaling_factor and
+                        data_offset allow linear corrections using the following convention:
+
+                        .. code-block::
+
+                                corrected_data = (data + offset) * scaling_factor
+
+                        This formula will derive the corrected value, when necessary.
+
+                        Use these fields to specify gain and/or pedestal constants that need to be applied
```

Comment:
```suggestion
Use these fields to specify gain and/or offset constants that need to be applied
```

Reply:
Happy to add more clarification in a different PR after this vote.